src/quo2/components/input/style.cljs

```diff
@@ -0,0 +1,170 @@
+(ns quo2.components.input.style
+  (:require [quo2.components.markdown.text :as text]
+            [quo2.foundations.colors :as colors]))
+
+(def variants-colors
+  "Colors that keep the same across input's status change"
+  {:light      {:label         colors/neutral-50
+                :icon          colors/neutral-50
+                :cursor        (get-in colors/customization [:blue 50])
```

Comment:
I think you can just use `(colors/custom-color :blue 50)` instead? 🤔

Reply:
Solved in 85df69cd70eae0fdb65751223e129487ec169f2c